### PR TITLE
generate clientsets, informers and listers since keda 1.5.0 does not do it

### PR DESCRIPTION
**Pull request by vaikas**
I'm trying to get rid of the internal directories and vendor in the proper way in eventing-rabbitmq. For that, I'll also need the injection, which this package gives. If we could also generated the pkg/client* here in a way that knative expects, would make it easier in that repo to reuse.

### CODE REVIEW

**Review by n3wscott — COMMENTED** [review]
/lgtm
/approve

File: hack/keda-register.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	GroupName = "keda.k8s.io"
```
Comment (zroubalik) [inline hack/keda-register.go:23]: @vaikas I was targeting this against KEDA v2, which will use `keda.sh` instead of `keda.k8s.io`

Comment (vaikas) [reply hack/keda-register.go:23]: ack, whats the expected timeline for that (keda getting to v2)? I'm just trying to find a way to make progress towards the eventing-rabbitmq which uses (currently v1.4 of keda looks like, which seems to require k8s 1.14??). Thoughts? Want me to change this to keda.sh? And is keda v2 stable enough to start using that instead? If so, I can just switch to that :)

Comment (zroubalik) [reply hack/keda-register.go:23]: @vaikas KEDA v2 Beta is about to be released (I hope Tomorrow). Then it is quite ready to use for development. KEDA 2.0 Final release should follow after a few weeks if there are no major problems with the Beta

Comment (zroubalik) [reply hack/keda-register.go:23]: I'll update deps and the relevant code in this repo once KEDA v2 Beta is out

**Pull request merged**